coral/types/api.d.ts

```diff
@@ -868,6 +871,60 @@ export type components = {
       connectorName: string;
       env: string;
     };
+    AclApproval: {
+      /** Format: int32 */
+      approvalId?: number;
+      /** @enum {string} */
+      approvalType?: "RESOURCE_TEAM_OWNER" | "ACL_TEAM_OWNER" | "TEAM";
```

Comment:
Do we have to update here as well ? RESOURCE_TEAM_OWNER to TOPIC_TEAM_OWNER ?

Reply:
Yes! Sorry missed this updating now!